python/pyarrow/serialization.pxi

```diff
@@ -88,17 +88,26 @@ cdef class SerializationContext:
             self.custom_deserializers[type_id] = custom_deserializer
 
     def _serialize_callback(self, obj):
-        if type(obj) not in self.type_to_type_id:
+        FOUND = False
```

Comment:
Can you use camel_case naming convention for this? UPPER_CASE is typically constants.

Reply:
Fixed and force-pushed